js/ssai_heartbeat_ooyala.js

```diff
@@ -76,6 +73,20 @@ OO.plugin('heartbeat', (OO) => {
       });
     }
 
+    /**
+     * Sets outer scope variable to a timer.
+     * Stops another timer if has been working before.
+     */
+    function startHeartBeat() {
+      stopHeartBeat();
+      heartbeatTimer = setInterval(reportHeartBeat, config.Interval);
+    }
+
+    /**
+     * Parse Guid.
+     * @param {string} url The streamUrl.
+     * @returns {''|string} Returns '' or first match result.
```

Comment:
`''` is a string too
The description could be more verbose
`@returns {string} Guid extracted from URL or empty string`